lib/mercadopago.rb

```diff
@@ -299,6 +299,7 @@ def initialize(debug_logger=nil)
 			if API_BASE_URL.scheme == "https" # enable SSL/TLS
 				@http.use_ssl = true
 				@http.ssl_options = OpenSSL::SSL::OP_NO_SSLv3 # explicitly tell OpenSSL not to use SSL3
+				@http.ssl_version = :TLSv1

```

Comment:
My app crashed with this error
``` SSL_connect returned=1 errno=0 state=SSLv3 read server hello A: sslv3 alert handshake failure```
And I can't sent any payment, that line solve my problem.